x-pack/filebeat/input/websocket/config_test.go

```diff
@@ -54,17 +54,6 @@ var configTests = []struct {
 		},
 		wantErr: fmt.Errorf("failed to check program: failed compilation: ERROR: <input>:3:79: found no matching overload for '_?_:_' applied to '(bool, list(dyn), null)'\n |      \"events\": has(state.cursor) && inner_body.ts > state.cursor.last_updated ? \n | ..............................................................................^ accessing config"),
 	},
-	{
-		name: "invalid_regexps",
-		config: map[string]interface{}{
-			"regexp": map[string]interface{}{
-				"products":  "(?i)(xq>)d+)",
-				"solutions": "(?)(Sws>(d+)",
-			},
-			"url": "wss://localhost:443/v1/stream",
-		},
-		wantErr: fmt.Errorf("failed to check regular expressions: error parsing regexp: unexpected ): `(?i)(xq>)d+)` accessing config"),
```

Comment:
Is the root cause of the flakyness that there are two possible errors that can occur given that both regular expressions are invalid? Rather than removing the test entirely it seems like better solution would be to address the non-determinism by removing `solutions`.

Reply:
@andrewkroh Yes absolutely, initial route was immature, have updated the test accordingly, and now only one specific failure should happen.